test/e2e.method.signing.js

```diff
@@ -253,6 +253,7 @@ describe('transaction and message signing [ @E2E ]', function() {
             value:    web3.utils.toHex(web3.utils.toWei('0.1', 'ether')),
             gas: web3.utils.toHex(21000),
             maxFeePerGas: '0x59682F00', // 1.5 Gwei
+            maxPriorityFeePerGas: '0x1DCD6500', // .5 Gwei
```

Comment:
Have to change this value because with change in this PR default value of `maxPriorityFeePerGas` is set to `2.5 Gwei` and `maxPriorityFeePerGas` can't be greater than `maxFeePerGas`.